### ISSUE TITLE
Error | Healer::Actions: | uninitialized constant

### ISSUE THREAD

**Issue by sflynn-anet** [pavel-odintsov/fastnetmon#512]
I am getting this error and am not sure exactly why.  It appears that I have everything setup (somewhat) but i'm not able to track this one down.

```
2016-04-29 15:07:59 - NameError - uninitialized constant Healer::Actions:
        /opt/nethealer/routes/ddos.rb:90:in `block (2 levels) in <class:Healer>'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1610:in `call'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1610:in `block in compile!'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:974:in `block (3 levels) in route!'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:993:in `route_eval'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:974:in `block (2 levels) in route!'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1014:in `block in process_route'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1012:in `catch'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1012:in `process_route'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:972:in `block in route!'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:971:in `each'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:971:in `route!'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1084:in `block in dispatch!'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:in `block in invoke'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:in `catch'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:in `invoke'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1081:in `dispatch!'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:906:in `block in call!'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:in `block in invoke'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:in `catch'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:in `invoke'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:906:in `call!'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:894:in `call'
        /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/xss_header.rb:18:in `call'
        /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/path_traversal.rb:16:in `call'
        /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/json_csrf.rb:18:in `call'
        /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/base.rb:49:in `call'
        /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/base.rb:49:in `call'
        /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/frame_options.rb:31:in `call'
        /var/lib/gems/2.3.0/gems/rack-1.6.0/lib/rack/nulllogger.rb:9:in `call'
        /var/lib/gems/2.3.0/gems/rack-1.6.0/lib/rack/head.rb:13:in `call'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:181:in `call'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:2021:in `call'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1486:in `block in call'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1795:in `synchronize'
        /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1486:in `call'
        /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/configuration.rb:82:in `call'
        /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/server.rb:507:in `handle_request'
        /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/server.rb:375:in `process_client'
        /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/server.rb:262:in `block in run'
        /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/thread_pool.rb:104:in `block in spawn_thread'
```

### Environment Configuration File

```
# cat .env
SSL_KEY_PATH="/opt/nethealer/ssl/nethealer.key"
SSL_CERT_PATH="/opt/nethealer/ssl/nethealer.crt"

NETHEALER_SERVER="${ipaddress}"
NETHEALER_INFLUXDB="${ipaddress}"
NETHEALER_USERNAME="fastnetmon"
NETHEALER_PASSWORD="fastnetmon"

GRAFANA_URL="http://${grafana_host}:3000"

THRESHOLD_EXPIRE="450"
THRESHOLD_WARNING="2"
THREDHOLD_CRITICAL="3"

NOTIFICATION_EMAIL_SMTP="localhost"
NOTIFICATION_EMAIL_FROM="root"
NOTIFICATION_EMAIL_TO="root"
NOTIFICATION_LOCATION="NULL"

#OPTIONAL PLUGINS

#PAGERDUTY_KEY=""
#FLOWDOCK_TOKENS="allow,multiple,flows,separated,by,comma"
```

**Comment by pavel-odintsov** [comment]
Hello!

Looks like you want to send this report to NetHealer's repository? :(

On Friday, 29 April 2016, sflynn-anet <notifications@github.com> wrote:

> I am getting this error and am not sure exactly why. It appears that I
> have everything setup (somewhat) but i'm not able to track this one down.
>
> 2016-04-29 15:07:59 - NameError - uninitialized constant Healer::Actions:
> /opt/nethealer/routes/ddos.rb:90:in block (2 levels) in <class:Healer>'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1610:incall'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1610:in block
> in compile!'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:974:inblock (3
> levels) in route!'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:993:in
> route_eval'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:974:inblock (2
> levels) in route!'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1014:in block
> in process_route'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1012:incatch'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1012:in
> process_route'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:972:inblock in
> route!'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:971:in each'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:971:inroute!'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1084:in block
> in dispatch!'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:inblock
> in invoke'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:in catch'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:ininvoke'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1081:in
> dispatch!'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:906:inblock in
> call!'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:in block
> in invoke'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:incatch'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1066:in invoke'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:906:incall!'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:894:in call'
>
> /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/xss_header.rb:18:in
> call'
> /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/path_traversal.rb:16:in
> call'
>
> /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/json_csrf.rb:18:in
> call'
> /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/base.rb:49:in
> call'
>
> /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/base.rb:49:in
> call'
> /var/lib/gems/2.3.0/gems/rack-protection-1.5.3/lib/rack/protection/frame_options.rb:31:in
> call'
> /var/lib/gems/2.3.0/gems/rack-1.6.0/lib/rack/nulllogger.rb:9:incall'
> /var/lib/gems/2.3.0/gems/rack-1.6.0/lib/rack/head.rb:13:in call'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:181:incall'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:2021:in call'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1486:inblock
> in call'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1795:in
> synchronize'
> /var/lib/gems/2.3.0/gems/sinatra-1.4.6/lib/sinatra/base.rb:1486:incall'
> /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/configuration.rb:82:in call'
> /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/server.rb:507:in
> handle_request'
> /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/server.rb:375:in
> process_client'
> /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/server.rb:262:inblock in
> run'
> /var/lib/gems/2.3.0/gems/puma-2.11.1/lib/puma/thread_pool.rb:104:in `block
> in spawn_thread'
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/pavel-odintsov/fastnetmon/issues/512>
>


-- 
Sincerely yours, Pavel Odintsov

**Comment by sflynn-anet** [comment]
Yes, i'm sorry --- was on the wrong browser tab.   :)

**Comment by pavel-odintsov** [comment]
No problem;)

On Friday, 29 April 2016, sflynn-anet <notifications@github.com> wrote:

> Closed #512 <https://github.com/pavel-odintsov/fastnetmon/issues/512>.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/pavel-odintsov/fastnetmon/issues/512#event-646110070>
>


-- 
Sincerely yours, Pavel Odintsov
